internal/imhash/imhash_test.go

```diff
@@ -62,3 +62,54 @@ func TestAppHash(t *testing.T) {
 	is.Equal(len(hash2), 11)
 	is.True(hash1 != hash2)
 }
+
+func TestAppHashEmbed(t *testing.T) {
+	is := is.New(t)
+	ctx := context.Background()
+	dir := t.TempDir()
+	td := testdir.New(dir)
+	td.Files["controller/controller.go"] = `
+		package controller
+		import "embed"
+		//go:embed *.sql
```

Comment:
Would a list of embeds work as well? For example,

```go
//go:embed image/* template/*
//go:embed html/index.html
var content embed.FS
```

Source: https://pkg.go.dev/embed#hdr-Directives

If not, maybe we could add that to this PR or leave a more specific TODO to support a list of embeds. What do you think?

Reply:
Looks like it works as-is; I've expanded the test to cover more cases.